### PR TITLE
docs: add note for sgx&vcl

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Loong Dai <loong.dai@intel.com>

Add note section about "not support".

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA] #22443
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/configuration/http/http_filters/sxg_filter.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@ If there is an error generating the SXG package we fall back to the original HTM
 
 For more information on Signed HTTP Exchanges see `this doc <https://developers.google.com/web/updates/2018/11/signed-exchanges>`_.
 
+.. note::
+
+ These features are not supported on Windows.
```
[the remark this answers is not in the document]
Comment (daixiang0) [reply docs/root/configuration/http/http_filters/sxg_filter.rst]: I have updated it, SXG does not provide its support info, as I know, it only supports amd Linux.

File: docs/root/configuration/other_features/vcl.rst [hunk 2]
```diff
@@ -9,6 +9,11 @@ VCL Socket Interface
 
   The VCL socket interface extension is experimental and is currently under active development.
 
+.. note::
+
+ These features are not supported on Windows, details refer to `VPP Supported Architectures <https://s3-docs.fd.io/vpp/22.10/aboutvpp/supported.html>`_.
```
Comment (phlax) [inline docs/root/configuration/other_features/vcl.rst:14]:
```suggestion
 These features are not supported on Windows, for details please refer to `VPP Supported Architectures <https://s3-docs.fd.io/vpp/22.10/aboutvpp/supported.html>`_.
```

File: docs/root/configuration/http/http_filters/sxg_filter.rst [hunk 3]
```diff
@@ -25,6 +25,11 @@ If there is an error generating the SXG package we fall back to the original HTM
 
 For more information on Signed HTTP Exchanges see `this doc <https://developers.google.com/web/updates/2018/11/signed-exchanges>`_.
 
+.. note::
+
+ These features are only supported on Linux amd64.
+
+
```
Comment (phlax) [inline docs/root/configuration/http/http_filters/sxg_filter.rst:32]: unnecessary ws

**Review by phlax — COMMENTED** [review]
thanks @daixiang0, a couple of minor nits and this should be gtm

File: docs/root/configuration/http/http_filters/sxg_filter.rst (hunk 3 above)
Comment (daixiang0) [reply docs/root/configuration/http/http_filters/sxg_filter.rst]: Deleted.

**Review by phlax — APPROVED** [review]
lgtm, thanks @daixiang0

**Pull request merged**